src/plugins/raven/components/active-edits-table/index.jsx

```diff
@@ -16,84 +16,70 @@ import delegate from './delegate';
 import selector from './selector';
 import TableRow from './TableRow';
 
-class ActiveEditsTable extends React.Component {
-    static propTypes = {
-      accessToken: PropTypes.string.isRequired,
-      collaborationNodes: PropTypes.arrayOf(PropTypes.instanceOf(Message)).isRequired,
-      handleUpdateCollaborations: PropTypes.func.isRequired,
-      title: PropTypes.string,
-    };
+export const ActiveEditsTable = ({
+  accessToken,
+  collaborationNodes,
+  handleUpdateCollaborations,
+  title,
+}) => {
+  const [status, setStatus] = useState(null);
+  const handleRefresh = async () => {
+    setStatus(STATUS_PENDING);
+    await handleUpdateCollaborations(accessToken);
+    setStatus(STATUS_FULFILLED);
+  };
 
-    static defaultProps = {
-      title: 'Active Edits',
-    }
+  useEffect(() => {
+    handleRefresh();
+  }, []);
 
-    constructor(props) {
-      super(props);
-      this.handleRefresh = this.handleRefresh.bind(this);
-      this.state = { intervalId: null, status: null };
-    }
+  const memoizedTableRows = useMemo(() => collaborationNodes.map((node, idx) => (
```

Comment:
@hunyoboy memoized isn't really needed here, optimized in this area is not important.

Reply:
@gdbrown yes the purpose of this memo is so the TableRows won't get auto-refresh when new collaborated nodes are added/removed and detected by Raven. i thought it won't but i noticed Raven will still auto-refresh the table. with this, it will keep using the memoized TableRow(s) until a change in status due to button click occurs.